examples/container-service-extension/v4.1/install/step1/3.11-cse-install-2-cse-server-prerequisites.tf

```diff
@@ -0,0 +1,281 @@
+# ------------------------------------------------------------------------------------------------------------
+# CSE v4.1 installation, step 1:
+#
+# * Please read the guide present at https://registry.terraform.io/providers/vmware/vcd/latest/docs/guides/container_service_extension_install
+#   before applying this configuration.
+#
+# * The installation process is split into two steps as the first one creates a CSE admin user that needs to be
+#   used in a "provider" block in the second one.
+#
+# * This file contains the same resources created by the "Configure Settings for CSE Server > Set Up Prerequisites" step in the
+#   UI wizard.
+#
+# * Rename "terraform.tfvars.example" to "terraform.tfvars" and adapt the values to your needs.
+#   Other than that, this snippet should be applied as it is.
+#   You can check the comments on each resource/data source for more help and context.
+# ------------------------------------------------------------------------------------------------------------
+
+# This is the RDE Interface required to create the "VCDKEConfig" RDE Type.
+# This should not be changed.
+resource "vcd_rde_interface" "vcdkeconfig_interface" {
+  vendor  = "vmware"
+  nss     = "VCDKEConfig"
+  version = "1.0.0"
+  name    = "VCDKEConfig"
+}
+
+# This resource will manage the "VCDKEConfig" RDE Type required to instantiate the CSE Server configuration.
+# The schema URL points to the JSON schema hosted in the terraform-provider-vcd repository.
+# This should not be changed.
+resource "vcd_rde_type" "vcdkeconfig_type" {
+  vendor        = "vmware"
+  nss           = "VCDKEConfig"
+  version       = "1.1.0"
+  name          = "VCD-KE RDE Schema"
+  schema_url    = "https://raw.githubusercontent.com/adambarreiro/terraform-provider-vcd/upgrade-cse41/examples/container-service-extension/v4.1/schemas/vcdkeconfig-type-schema-v1.1.0.json"
```

Comment:
(Reminder?) The link points to a personal repo.

Reply:
Changed now (time to merge 😄 )